bitrise/setup.go

```diff
@@ -46,9 +46,14 @@ var PluginDependencyMap = map[string]PluginDependency{
 }
 
 func RunSetupIfNeeded(logger log.Logger) error {
-	if !configs.CheckIsSetupWasDoneForVersion(version.VERSION) {
-		log.Warnf("Setup was not performed for this version of bitrise, doing it now...")
-		return RunSetup(logger, version.VERSION, SetupModeDefault, false)
+	versionMatch, setupVersion := configs.CheckIsSetupWasDoneForVersion(version.VERSION)
+	if setupVersion == "" {
+		log.Warnf("No setup was done yet, running setup now...")
```

Comment:
Why is this message a warning? I think these cases aren't indicating any issue, that might cause a failure or problem later on.
Also, I don't think this is very important info for all the builds, I suggest normal log level.

Reply:
True, changed both of them to `Infof`